marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/suiteSetup.sjs

```diff
@@ -56,13 +61,14 @@ xdmp.documentInsert("/exp/doc1",
               }
             ],
             "searchEntityProp2": "doc1SrchEntyProp2",
-            "searchEntityProp1": "doc1SrchEntyProp1"
+            "searchEntityProp1": "doc1SrchEntyProp1",
```

Comment:
@ryanjdew  @rahulvudutala Is the reason these documents are defined inline instead of in the "./test-data/content" folder because of the need to define metadata and collections for the documents? If so, I'll think about a way to define those in the test documents. I'm thinking we could do a custom JSON structure with properties of "document", "metadata", and "collections.